ui/src/message_popup/scaffolding_card_test.jsx

```diff
@@ -0,0 +1,56 @@
+/* @flow weak */
+import React from 'react';
+
+import {shallow} from 'enzyme';
+import sinon from 'sinon';
+import {expect} from 'chai';
+import * as TestFixtures from './test_fixtures.js';
+
+import ScaffoldingCard from './scaffolding_card.jsx';
+import Toggle from 'material-ui/Toggle';
+import Slider from 'material-ui/Slider';
+
+
+function testProps(props) {
+  return {
+    scaffolding: TestFixtures.practiceScaffolding,
+    initialEmail: 'user@test.com',
+    query: {},
+    onSessionConfigured: sinon.spy(),
+    ...props
+  };
+}
+
+function expectCoreOptions(wrapper) {
+  expect(wrapper.text()).to.contain('Practice scenarios:');
+  expect(wrapper.find('RadioButtonGroup[name="indicatorId"]').length).to.equal(1);
```

Comment:
yeah, just noticed it!  https://github.com/airbnb/enzyme/blob/master/docs/api/selector.md